ci/build-test-qemu.yaml

```diff
@@ -0,0 +1,37 @@
+job:
+  strict: true
+recipe:
+  git_ref: master
+  git_url: https://github.com/openshift/os
+  repos:
+  - url: http://base-4-8-rhel84.ocp.svc.cluster.local
```

Comment:
Totally fine as is but I think down the line it'd be nicer to have some magic that e.g. parses the manifest to find the expected repos or so.

Reply:
:100: I agree.